### PR TITLE
Dashboardpage

### PR DESCRIPTION
**Pull request by jnguy2018**
Made it so when we add/remove a student from a course, it also removes the course from the 'courseIds' array in the student object

### CODE REVIEW

File: routes/api/courses.js [hunk 1]
```diff
@@ -121,7 +123,7 @@ router.put("/:courseId/delete/:studentId", passport.authenticate('jwt', {session
   if(!req.params.studentId) return res.status(400).json({ status: "failure", error: "missing studentId" })
   
   if(!await isOwner(Course, req.params.courseId, req.user._id.toString(), "professorId")){
-    return res.status(401).send("Unauthorized")
+    return res.status(401).json({status:"fail"})
```
Comment (ChrisLouie10) [inline routes/api/courses.js:126]: why did you change this to a json?

File: routes/api/courses.js [hunk 2]
```diff
@@ -99,6 +99,8 @@ router.put("/:courseId/:studentId", passport.authenticate('jwt', {session: false
         error: "student not found" 
       })
     } else {
+      student.courseIDs.push(courseId);
+      student.save();
```
Comment (ChrisLouie10) [inline routes/api/courses.js:103]: should move this til after you find the course or else you could be adding courses that don't exist.

**Review by ChrisLouie10 — CHANGES_REQUESTED** [review]

File: routes/api/courses.js (hunk 1 above)
Comment (jnguy2018) [reply routes/api/courses.js]: oops was testing smt
